todos/src/supabaseData.js

```diff
@@ -0,0 +1,65 @@
+import { createClient } from "@supabase/supabase-js";
+
+const projectURL = "https://aemdlybecnilhqobaiwn.supabase.co";
+const anonKey =
+  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImFlbWRseWJlY25pbGhxb2JhaXduIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NzYyNjE0NDQsImV4cCI6MTk5MTgzNzQ0NH0.MawqvRVONrJWmOTjAmkSoBKLMPAl0dyiw2k2nUEt3IE";
+
+const supabase = createClient(projectURL, anonKey);
```

Comment:
Can we create singleton class so that supabase doesn't get initiated multiple time?

Reply:
Done Bhai. Separated in another .js file.